pkgs/by-name/ta/tailspin/package.nix

```diff
@@ -16,6 +16,11 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-ohfza2ti7Ar/9TV/WoTL5g6CPaONrxtr7nW0qmLdB/8=";
 
+  postPatch = ''
+    substituteInPlace tests/utils.rs --replace \
```

Comment:
`--replace` by itself is deprecated. use `--replace-fail` for instance

Reply:
Fixed, thanks